src/index.js

```diff
@@ -76,6 +123,8 @@ const letters = 'bBdDgGtTvV'.split("");
 const control_before = Math.round(Math.random()); // 0 control comes before test, 1, after
 let block_acc = 0; // record block accuracy to determine next blocks delay
 let delay = 1; // starting delay
+let previous_delay = -1;
```

Comment:
[nit]: Can you add a comment for the reason of using "previous_delay"?

We can just say something like _used to find if the level went up or down_.